src/test/librados/testcase_cxx.cc

```diff
@@ -389,10 +389,10 @@ void RadosTestECPP::SetUp()
   nspace = get_temp_pool_name();
   ioctx.set_namespace(nspace);
   bool req;
-  ASSERT_EQ(0, ioctx.pool_requires_alignment2(&req));
-  ASSERT_TRUE(req);
-  ASSERT_EQ(0, ioctx.pool_required_alignment2(&alignment));
-  ASSERT_NE(0U, alignment);
+  //ASSERT_EQ(0, ioctx.pool_requires_alignment2(&req));
```

Comment:
is the back and forth on the commenting out of these lines required?

Reply:
@ronen-fr pushed that too soon, i fixed that.